skills/ship/epic-runner/SKILL.md

```
@@ -461,42 +525,117 @@ the PR for a person rather than failing it.
+integrated result. Send it back to an issue-agent on the existing branch to merge the epic
+tip in and take the PR to green again — that CI run is the first thing to have exercised the
+integrated head. On a protected target this is what `STRICT_POLICY: required` would enforce;
+an `epic/*` branch is usually unprotected, so nothing else does.
+
+With the predicate satisfied, authorize the merge by dispatching `MERGE`. The reread and the
+merge live in one context precisely because nothing may come between them:
 
-```bash
-git fetch "$BASE_REMOTE" "epic/$SLUG"
-git merge-base --is-ancestor "$BASE_REMOTE/epic/$SLUG" "$APPROVED_SHA"
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)